src/web_interface/templates/generic_view/file_object_macros.html

```diff
@@ -13,7 +13,7 @@ <h6>{{ file_object.current_virtual_path[0] | replace_uid_with_hid(root_uid=root_
                 , Type: {{ file_object['mime-type'] }}
             {% endif %}
         </span>
-        <a href="location.href='/download/{{ file_object.uid }}'"><i class="fas fa-download"></i></a>
+        <a href="#" onclick="location.href='/download/{{ file_object.uid }}'"><i class="fas fa-download"></i></a>
```

Comment:
This should be even simpler (I don't know why there was a `location.href` in a hyperlink tag in the first place):
```suggestion
        <a href="/download/{{ file_object.uid }}"><i class="fas fa-download"></i></a>
```